server/models/handlers/school.js

```diff
@@ -1,8 +1,10 @@
-// Create stub which returns an array of Worker ID's given an input of school ID or school Name
+const WorkerObjectModel = require('../db/school');
```

Comment:
Hi Simran,
For clearer code readability, I would suggest renaming this variable to be schoolModel as the file appears to be using school rather than worker.

Reply:
Hi Amy! Thank you for your comment! I've changed the variable to be schoolModel as requested!